### DISCUSSION TITLE
Unable to change github account on git w/ windows 11

### DISCUSSION THREAD

**Discussion by Sekitan** [community/community#32762] [Other Feature Feedback, Questions, & Ideas] [answered]
I can't change my github account when I push something to a repository.  I changed my git --global username and email, deleted the git entry in windows credentials manager, I even tried blocking the unwanted account from my account.  No matter what when I push to remote the commit is made under the old account and not the new one.

EDIT: Fixed it by deleting the .git folder from my local folder lmao

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for sharing your fix!
